FYDP_ModV1/src/version1.java

```diff
@@ -1,14 +1,181 @@
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Iterator;
+
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.CellType;
+import org.apache.poi.ss.usermodel.CellValue;
+import org.apache.poi.ss.usermodel.FormulaEvaluator;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.xssf.usermodel.XSSFSheet;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+import static org.apache.poi.ss.usermodel.Row.MissingCellPolicy;
+
 public class version1 {
 
-	public static void main(String[] args) {
+	public static void main(String[] args) throws IOException {
 		// TODO Auto-generated method stub
 		modelConfig();
 	}
 	
-	public static void modelConfig() {
+	public static void modelConfig() throws IOException {
+	//start of excel read in
+		// write your code here
+        //read
+        String excelFilePath = "/Users/mccurdy/Documents/4B/FYPD/excel_read_in/Jan-30-Front-End.xlsx";
+        FileInputStream inputStream = new FileInputStream(new File(excelFilePath));
+        Workbook workbook = new XSSFWorkbook(inputStream);
+        int numberOfSheets = workbook.getNumberOfSheets();
+        ArrayList<String> readData = new ArrayList<String>(0);
+        //number of teachers
+        int n2 = -1;
+        //list of teachers names
+        ArrayList<String> teacherNames = new ArrayList<String>(0);
+        //teacher allocation
+        ArrayList<Double> FTE = new ArrayList<Double>(0);
+        //Name of schedule
+        String schedule_name = "empty";
+
+        //Individual teacher allocation time staple
+        ArrayList<int[]> availableTime = new ArrayList<int[]>();
+        //number of french teachers
+        //index of first french teacher 
+        int frenchTeachlb = -1;
+        //change to first sheet numberOfSheets to 1
+        for (int i = 0; i < 1; i++) {
+            Sheet sheetIndex = workbook.getSheetAt(i);
+            //getting schedule title
+            int rowStart = sheetIndex.getFirstRowNum();
+            Row r = sheetIndex.getRow(rowStart);
+            int sheetTitleStart = 4; 
+            Cell titlecell = r.getCell(sheetTitleStart);
+            if (titlecell == null || titlecell.getRichStringCellValue().getString() == "") {
+            	Date today = Calendar.getInstance().getTime();
+            		schedule_name = "Generated_Schedule " + today;
+             } else {
+            	 	schedule_name = titlecell.getRichStringCellValue().getString();
+             }
+        
+            //get teacher names, allocation, and fte 
+            //index of start of teacher matrix
+            int teacher_matrix_start = 6;
+            //teacher name col
+            int teacher_name_row = 0;
```

Comment:
can you change these variable names to col? I'm pretty sure they are referencing columns not rows